htdocs/fichinter/card.php

```diff
@@ -1525,7 +1525,7 @@
 					print '<td align="right">';
 					$selectmode='select';
 					if (! empty($conf->global->INTERVENTION_ADDLINE_FREEDUREATION)) $selectmode='text';
-					$form->select_duration('duration',$objp->duree,0, $selectmode);
+					$form->select_duration('duration',$objp->duree, $selectmode);
```

Comment:
I am really confused in what happened in my repository.
It seems that I merged accidentally some temporary branches.
I do not knoy how to resolve it.
Can you do it for me ?

2015-11-28 22:40 GMT+01:00 Abbes Bahfir <bafbes@gmail.com>:

> It's just an error in my repository.
> It's corrected now.
>
> 2015-11-28 15:52 GMT+01:00 Laurent Destailleur <notifications@github.com>:
>
>> In htdocs/fichinter/card.php
>> <https://github.com/Dolibarr/dolibarr/pull/4070#discussion_r46080525>:
>>
>> > @@ -1525,7 +1525,7 @@
>> >  					print '<td align="right">';
>> >  					$selectmode='select';
>> >  					if (! empty($conf->global->INTERVENTION_ADDLINE_FREEDUREATION)) $selectmode='text';
>> > -					$form->select_duration('duration',$objp->duree,0, $selectmode);
>> > +					$form->select_duration('duration',$objp->duree, $selectmode);
>>
>> This looks to be a regression.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/Dolibarr/dolibarr/pull/4070/files#r46080525>.
>>
>
>